assistant_dists/dream_kg/docker-compose.override.yml

```diff
@@ -256,63 +220,6 @@ services:
           memory: 128M
         reservations:
           memory: 128M
-  
-  transformers-lm-gptjt:
-    env_file: [ .env ]
-    build:
-      args:
-        SERVICE_PORT: 8161
-        SERVICE_NAME: transformers_lm_gptjt
-        PRETRAINED_MODEL_NAME_OR_PATH: togethercomputer/GPT-JT-6B-v1
-        HALF_PRECISION: 1
-      context: .
-      dockerfile: ./services/transformers_lm/Dockerfile
-    command: flask run -h 0.0.0.0 -p 8161
-    environment:
-      - CUDA_VISIBLE_DEVICES=0
-      - FLASK_APP=server
-    deploy:
-      resources:
-        limits:
-          memory: 50G
-        reservations:
-          memory: 50G
-
-  dff-dream-persona-gpt-jt-prompted-skill:
```

Comment:
why do you remove those skills? there will no be any skill to generate responses after this removal

Reply:
We're adding the distribution's own skills in [this PR](https://github.com/deeppavlov/dream/pull/361) which will be merged later on